### DISCUSSION TITLE
关于账号切换的一些疑问

### DISCUSSION THREAD

**Discussion by DLmaster361** [MaaAssistantArknights/MaaAssistantArknights#10814] [Q&A]
B服账号切换的查询逻辑究竟是怎样的？在推荐分辨率下还能做什么以保证账号切换的成功率？

**Comment by DLmaster361** [comment]
另：当前在1080p与720p都无法正确切换账号

**Reply by Daydreamer114** [reply]
不支持b服吧我记得

**Comment by DLmaster361** [comment]
参见https://maa.plus/docs/zh-cn/manual/introduction/startup.html，至少在以前是支持的

**Reply by Daydreamer114** [reply]
见 https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/10815#issuecomment-2413019161

**Reply by ABA2396** [reply]
pr加了b服支持，测测看 https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/10817
